SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorCloudLauncherSettings.cpp

```diff
@@ -112,9 +129,20 @@ void USpatialGDKEditorCloudLauncherSettings::SetNumberOfSimulatedPlayers(uint32
 
 bool USpatialGDKEditorCloudLauncherSettings::IsDeploymentConfigurationValid() const
 {
-	return IsAssemblyNameValid(AssemblyName) &&
+	bool result = IsAssemblyNameValid(AssemblyName) &&
 		IsDeploymentNameValid(PrimaryDeploymentName) &&
 		IsProjectNameValid(ProjectName) &&
 		!SnapshotPath.FilePath.IsEmpty() &&
-		!PrimaryLaunchConfigPath.FilePath.IsEmpty();
+		!PrimaryLaunchConfigPath.FilePath.IsEmpty() &&
+		IsRegionCodeValid(PrimaryDeploymentRegionCode);
+
```

Comment:
Could we prematurely return false here if the primary deployment config is invalid or is there a reason we still need to check whether the sim player deployment fields are valid?

Reply:
Alternatively we can just make `if (IsSimulatedPlayersEnabled())` to `if (result && IsSimulatedPlayersEnabled())` for readability.

Reply:
Actually result will shortcircuit on line 141 if false, so no other checks will be executed in that case. I think its fine as is then